### ISSUE TITLE
scLink: Inferring Sparse Gene Co-expression Networks from Single-cell Expression Data

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#33]
## TL;DR

A system-level understanding of the regulation and coordination mechanisms of gene expression is essential for studying the complexity of [biological processes](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/biological-phenomena-and-functions-concerning-the-entire-organism) in health and disease. With the rapid development of single-cell [RNA sequencing](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/rna-sequence) technologies, it is now possible to investigate gene interactions in a cell type-specific manner. Here we propose the scLink method, which uses statistical network modeling to understand the co-expression relationships among genes and construct sparse gene co-expression networks from single-cell [gene expression data](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/gene-expression-data). We use both simulation and real data studies to demonstrate the advantages of scLink and its ability to improve single-cell gene network analysis. The scLink R package is available at https://github.com/Vivianstats/scLink.

### Paper Link

https://www.sciencedirect.com/science/article/pii/S1672022921001455

### Author/Institution

Wei Vivian Li (Department of Biostatistics and Epidemiology, The State University of New Jersey)

## Cite

Wei Vivian Li, Yanzeng Li,
scLink: Inferring Sparse Gene Co-expression Networks from Single-cell Expression Data,
Genomics, Proteomics & Bioinformatics,
Volume 19, Issue 3,
2021,
Pages 475-492,
ISSN 1672-0229,
https://doi.org/10.1016/j.gpb.2020.11.006.
(https://www.sciencedirect.com/science/article/pii/S1672022921001455)
